codeworld-account/stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
This comment suggests that this shouldn't be checked in, if it's autogenerated?  Can we add it to `.gitignore`?  Same for others.

Reply:
Looks like it is optional to check in the `stack.yaml.lock` files: https://docs.haskellstack.org/en/stable/lock_files/#lock-files

Reply:
Okay, I see the point there, but I just don't agree in this context.  Since stack isn't the main build process, I'd rather reduce the maintenance load for it, and these are files that would have to be updated as things change.  So please remove `stack.yaml.lock` from the pull request, and add it to `.gitignore` instead?